.envrc.example

```diff
@@ -0,0 +1,6 @@
+export SECRET_KEY=codeforpoznan
+export DB_HOST=db
+export DB_PORT=5432
+export DB_NAME=cfp_v3
+export DB_USER=cfp_v3
+export DB_PASSWORD=cfp_v3
```

Comment:
I know it's an example but come on :D

Reply:
no, that's on purpose. If someone by mistake or for any other reason will copy the `envrc.example` to `.envrc` it will not break his environment, because the default variables are the same as defined in `docker-compose.yaml`.
I could put something else in here, but that would potentially break env if someone would follow the procedure from readme without replacing the secrets with something more realistic.

Reply:
In other words, it's a safeguard for a hypotetical user who would follow the whole procedure but failed to obtain the secrets from admins. If these values would be different then he would probably end up with broken DB connection.